src/rpsapp.js

```diff
@@ -0,0 +1,93 @@
+import getThrow from './rpstest.js';
+
+const throwButton = document.getElementById('throwbutton');
+
+ 
+const winCount = document.getElementById('win-count');
+const lossCount = document.getElementById('loss-count');
+const tiesCount = document.getElementById('ties-count');
+const computerImage = document.getElementById('computer-choice');
+const addMessage = document.getElementById('message');
+
+let moneyRemaining = 50;
+let updateMoney = document.getElementById('moneyremaining');
+let amount = document.getElementById('amount');
+let currentBet = 0;
+let bet = document.getElementById('bet');
+updateMoney.textContent = `YOU HAVE $${moneyRemaining} TO BET`;
+
+let wins = 0;
+let losses = 0;
+let ties = 0;
+
+let message = '';
+// throw rock, paper, or scissors, based off random number. Function onclick of a random number.
+bet.addEventListener('click', () => {
+    if(moneyRemaining >= 0) {
+        currentBet = Number(amount.value);
+    } else {
+        updateMoney.textContent = 'YOU CAN\'T BET ANYMORE MONEY, GO HOME!';
+        bet.disabled = true;
+        amount.disabled = true;
+    }
+});
+
+throwButton.addEventListener('click', () => {
+    const computerChoice = getThrow();
+    const userChoice = document.querySelector('input:checked').value;
+
+    if(userChoice === 'rock' && computerChoice === 'scissors' && moneyRemaining > 0) {
```

Comment:
Try to break this down into functions - you're repeating a lot of actions here, so refactoring to use more functions would help you lessen the amount of code you have. The logic here is correct, but I think your solution involves too much code.